### ISSUE TITLE
CLI hangs after test saved

### ISSUE THREAD

**Issue by arggh** [qawolf/qawolf#603]
Congrats on 1.0!

I just created a new test with the latest QA Wolf and noticed, that after I select "Save and exit" from the CLI, the program doesn't return to command line.

```
    🐺  QA Wolf is ready to create code!

      at ../node_modules/qawolf/src/create-code/create.ts:59:11

? Edit your code at: src/signup.test.js 💾  Save and exit


```

**Comment by jperl** [comment]
Thanks for reporting this, I will look into this issue. Does it happen every time, or only once in a while?

**Comment by jperl** [comment]
@arggh This should be fixed in qawolf@1.0.1. Please let me know if it is still an issue.
